.github/CODEOWNERS

```diff
@@ -307,3 +307,4 @@ reports/distributions/contrib.yaml @open-telemetry/collector-contrib-approvers
 
 exporter/skywalkingexporter/                                        @open-telemetry/collector-contrib-approvers 
 receiver/googlecloudspannerreceiver/                                @open-telemetry/collector-contrib-approvers 
+e2e-tests/
```

Comment:
Do we need this here ?

Reply:
Yeah there's a CI check that fails without this